pkg/kube/controllers/extendedjob/errand_reconciler.go

```diff
@@ -100,6 +110,13 @@ func (r *ErrandReconciler) Reconcile(request reconcile.Request) (reconcile.Resul
 			err = ctxlog.WithEvent(eJob, "UpdateError").Errorf(ctx, "Failed to traverse to 'trigger.strategy=done' on job '%s': %s", eJob.Name, err)
 			return reconcile.Result{Requeue: false}, err
 		}
+	} else {
+		// several updates to one resource result in 'object has been modified' and the error triggers a second reconcile even though Requeue was false!?
```

Comment:
Yes: https://github.com/cloudfoundry-incubator/cf-operator-vendor/blob/e337a0cdf3c31f7053be70212b3059d078ed8fa1/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go#L215

Only return empty error if you want to skip this reconciliation.